common/src/main/resources/assets/ping-wheel/lang/uk_ua.json

```diff
@@ -0,0 +1,44 @@
+{
+	"ping-wheel.key.ping-location": "Поставити мітку",
+	"ping-wheel.key.open-settings": "Відкрити налаштування",
+	"ping-wheel.key.name-labels": "Показувати ярлики імен",
+	"ping-wheel.subtitles.ping": "Мітка поставлена",
+
+	"ping-wheel.settings.pingVolume": "Гучність мітки: %s",
+	"ping-wheel.settings.pingDuration": "Тривалість мітки: %s",
+	"ping-wheel.settings.pingDistance": "Дистанція мітки: %s",
+	"ping-wheel.settings.correctionPeriod": "Період коригування: %s",
+	"ping-wheel.settings.itemIconVisible": "Піктограми предметів",
+	"ping-wheel.settings.directionIndicatorVisible": "Індикатор напрямку",
+	"ping-wheel.settings.nameLabelForced": "Завжди показувати імена",
```

Comment:
> I came up with a translation that is one letter shorter, but it distorts the meaning a bit:
Постійний показ імен
🤔

I think as long as it delivers the general idea, this should be fine.
Looks good in the menu:
![image](https://github.com/LukenSkyne/Minecraft-Ping-Wheel/assets/29918069/b566d565-f541-4161-b0f2-87e3f42a9672)

Reply:
Okey, let's go with this option!

Is it technically possible for mods to use scrolling text in buttons, like in the vanilla Minecraft settings? If so, it would be better to use the original version.